CHANGES.rst

```diff
@@ -35,6 +35,8 @@ Improvements
 - Hide the top box with the latest files of an editable until it has been accepted
   and published (:issue:`5660`, :pr:`5665`)
 - Allow uploading files when requesting changes on the editing timeline (:pr:`5612`)
+- Allow microsservices to mark new editables as ready-for-review without creating
```

Comment:
"Allow an editing service [check how we call it in the UI] ..."

(and i'd move it to the "internal changes" section)